operator/controllers/flowtest_controller.go

```diff
@@ -48,33 +55,63 @@ type FlowTestReconciler struct {
 func (r *FlowTestReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	logger := log.FromContext(ctx)
 
+	logger.Info("Reconciling")
```

Comment:
nit: This should be a debug statement or more detailed.

Reply:
```
2021-07-12T21:03:44.978+0530	INFO	controller-runtime.manager.controller.flowtest	Reconciling	{"reconciler group": "loggingplumber.isala.me", "reconciler kind": "FlowTest", "name": "flowtest-sample", "namespace": "default"}
```
Logger appends most metadata we need, is there anything else you want to be added specifically?

Reply:
it is more of the intent of the message.  reconciling is vauge, it is a reconcile loop it reconciles.  There is value in saying "change detected starting reconciliation" or "found flowtest with no status generating resources".  When you write log messages think of it from the standpoint of someone trying to understand what is happening without being able to see the code.

Reply:
k8s reconciliation loop is a bit tricky, I will open an issue for this for now and can we talk about this more in the next meeting?

Reply:
This will get addressed in https://github.com/MrSupiri/rancher-logging-explorer/issues/13